Resources/doc/index.rst

```diff
@@ -35,6 +35,15 @@ Register bundle into ``config/bundles.php`` (Flex did it automatically):
 Generate the SSL keys
 ~~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+    For the version 2.xx of this bundle, you can use Web-Token and generate
```

Comment:
A link to the project repository or website would be nice on `Web-Token`

Reply:
A link is now present